src/main/resources/db/changelog/changesets/changelog_20231004T104042Z.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:pro="http://www.liquibase.org/xml/ns/pro" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/pro http://www.liquibase.org/xml/ns/pro/liquibase-pro-latest.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-latest.xsd">
+    <changeSet author="millotflo (generated)" id="1696416067509-1">
+        <addColumn tableName="feeder_results">
+            <column name="feeder_result_uuid" type="uuid">
+                <constraints nullable="false"/>
```

Comment:
This will not work with old data in the table without value in this column it will proc an error.

Reply:
Yes you're right my bad, I push the correction (delete and rebuild all the tables)